packages/axelar-wasm-std/src/voting.rs

```diff
@@ -217,7 +217,10 @@ impl Poll for WeightedPoll {
 
         for (tally, vote) in self.votes.iter_mut().zip(votes.iter()) {
             if *vote {
-                *tally += self.snapshot.get_participant_weight(sender).unwrap();
+                *tally += self
```

Comment:
```
        for (tally, vote) in self.votes.iter_mut().zip(votes.iter()) {
            if *vote {
                *tally += Uint256::from(participant.weight);
            }
        }
```

Reply:
Agree with CJ, and I'd also do something like 
```
        self.votes
            .iter_mut()
            .zip(votes.iter())
            .filter(|(_, vote)| **vote)
            .for_each(|(tally, _)| {
                *tally += self.snapshot.get_participant_weight(sender).expect("msg")
            });
```

Reply:
combined 💡  from two of you